server/db/database.sql

```diff
@@ -2,10 +2,11 @@ CREATE TABLE posts (
     id SERIAL PRIMARY KEY,
     title VARCHAR(255),
     pseudonym VARCHAR(255),
-    body VARCHAR(255)
+    body VARCHAR(255),
+    date DATE
```

Comment:
why not make each one NOT NULL so each post is required to have them

Reply:
good idea. we added checks in the front end but adding it in the SQL would've been better